### PR TITLE
Correct links to Renovate CE Docker images

### PR DESCRIPTION
**Pull request by jamietanna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Please view [the markdown docs in this repository](https://github.com/mend/renov
 
 ## Download
 
-Mend Renovate is distributed via Docker Hub using the namespace [whitesource/renovate](https://hub.docker.com/r/whitesource/renovate).
+Mend Renovate is distributed via GitHub Container Registry using the namespace [mend/renovate-ce]([https://hub.docker.com/r/whitesource/renovate](https://github.com/orgs/mend/packages/container/package/renovate-ce)).
```
Comment (nabeelsaabna) [inline README.md:13]: any reason for keeping `https://hub.docker.com/r/whitesource/renovate` ?
```suggestion
Mend Renovate is distributed via GitHub Container Registry using the namespace [mend/renovate-ce](https://github.com/orgs/mend/packages/container/package/renovate-ce).
```

Comment (jamietanna) [reply README.md:13]: Nope, purely accidental :joy:

**Comment by nabeelsaabna** [comment]
I'll leave it to Rhys or Justin in case..

**Review by rarkins — APPROVED** [review]

**Comment by rarkins** [comment]
Thanks! Strange we missed that one

**Comment by jamietanna** [comment]
All good, appreciate it! Is there any guidance on whether folks should use the `-full` image?

**Pull request merged**